lib/ansible/modules/cloud/docker/docker_volume.py

```diff
@@ -223,7 +223,7 @@ def present(self):
         if self.existing_volume:
             differences = self.has_different_config()
 
-        if differences and self.parameters.force:
+        if differences or self.parameters.force:
```

Comment:
This looks like a fix that should come as a separate PR.

Reply:
Sure, I can move it into a separate PR. I'll then rebase this PR once that PR is merged (as otherwise the test won't work :) ).